### PR TITLE
Update container_bundle doc to remark deprecation of "stamp" attribute

### PR DESCRIPTION
**Pull request by chanseokoh**
When I tried using it, `bazel` says it is depreated.

`DEBUG: /.../home/chanseok/.cache/bazel/_bazel_chanseok/2fa25e6c02293c6f4b6133cec26037eb/external/io_bazel_rules_docker/container/bundle.bzl:42:9: Attr 'stamp' is deprecated; it is now automatically inferred. Please remove it from //:all`

### CODE REVIEW

**Comment by chanseokoh** [comment]
@nlopezgi well, the bot isn't helping; it says to look into `OWNERS`, which doesn't exist in this PR.

File: README.md [hunk 1]
```diff
@@ -1882,6 +1882,7 @@ A rule that aliases and saves N images into a single `docker save` tarball.
       <td><code>stamp</code></td>
       <td>
         <p><code>Bool; optional</code></p>
+        <p>Deprecated: it is now automatically inferred.</p>
```
Comment (xingao267) [inline README.md]: Thanks for the PR. Could you also update the doc for container_push?

Comment (chanseokoh) [reply README.md]: Oh, didn't know it's deprecated too. Updated `container_push` too.

**Pull request merged**